src/modelfree/simulation_utils.py

```diff
@@ -20,6 +20,23 @@ def simulate(env, agents, render=False):
 
         yield observations, rewards, dones, infos
 
+def simulate_single(env, agent, render=False):
```

Comment:
Do we really need a function for both multi-agent and single-agent simulation? :(

Perhaps make a wrapper that turns a (Vec)Env into a (Vec)MultiEnv with a single agent?